types/ApiResponses.ts

```diff
@@ -1,3 +1,4 @@
+import { ResumeModel } from 'models/ResumeModel';
```

Comment:
nit: this needs to be a relative URL instead of absolute, since our build system doesn't properly resolve absolute paths